src/DataStructures/Python/Tensor/TensorData.cpp

```diff
@@ -0,0 +1,49 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <string>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/TensorData.hpp"
+#include "Utilities/GetOutput.hpp"
+#include "Utilities/StdHelpers.hpp"
```

Comment:
I don't think you need this include (anymore), please delete

Reply:
Seems like this include is necessary for introducing `operator<<` for the `std::vector`s in the `get_output(extents_and_data.extents)` and `get_output(extents_and_data.tensor_components)` calls

Reply:
I see, ok then, thanks for checking